railties/lib/rails/engine/commands_tasks.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails/commands/rake_proxy'
```

Comment:
Has require a `rake` in the `rails/commands/rake_proxy`, I think here that it is not need.